app/_redirects

```diff
@@ -1,6 +1,7 @@
-# FREE TRIALS
+# FREE TRIALS, GENERAL LICENSING
 /free-trial/guide/                    https://konghq.com/install
-
+/gateway/plan-and-deploy/licenses     https://konghq.com/pricing/
```

Comment:
```suggestion
/gateway/plan-and-deploy/licenses     /gateway/latest/plan-and-deploy/licenses
```
This topic does still exist, and there's more info on it than just the plans&comparison: https://docs.konghq.com/gateway/2.6.x/plan-and-deploy/licenses/